tw-docops/tw-mkdocs/docs/index.md

```diff
@@ -1,38 +1,13 @@
-# Index Page
+# Libertad Pantoja
 
-## Wizeline DocOps Template
+## Technical Writer | Wizeline
 
-Welcome to the DocOps Template
+Libertad originally majored in Genomic Sciences where she was exposed to Big Data Analytics for the first time. After that, Libertad pursued a PhD in Biomedical Sciences. She previously worked in Data Science, focusing on banking data analytics. Her work also involved to transform the analytics portal in sata scientists friendly site.
 
-## Adding Information to the Page
+Libertad loves bringing a better understanding of the tools and processes that take place in the technological daily life. She aims to become a better writer (both in English and Spanish) of speculative fiction stories. In her free time Libertad enjoys doing yoga, spending time with her family, reading, drawing, and playing board games and videogames. 
```

Comment:
...In her free time, Libertad enjoys (...)

`Missing comma after In her free time`